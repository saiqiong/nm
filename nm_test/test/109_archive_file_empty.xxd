213c 6172 6368 3e0a 656d 7074 7920 2020
2020 2020 2020 2020 3135 3230 3435 3335
3832 2020 3130 3935 3520 3020 2020 2020
3130 3036 3434 2020 3020 2020 2020 2020
2020 600a                              
