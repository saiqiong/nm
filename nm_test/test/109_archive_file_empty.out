!<arch>
empty           1520453582  10955 0     100644  0         `
